src/components/TimeSliceGroup.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes, Component } from 'react'
+import SelectableTimeSlice from '../containers/SelectableTimeSlice.jsx'
+import date from '../utils/dates.js'
+
+export default class TimeSliceGroup extends Component {
+  static propTypes = {
+    slices: PropTypes.oneOf([2,3,4,5]),
+    step: PropTypes.number.isRequired,
+    value: PropTypes.instanceOf(Date).isRequired,
+    showlabels: PropTypes.bool,
+    isNow: PropTypes.bool,
+    timeGutterFormat: PropTypes.string,
+    timesliceClassnames: PropTypes.string,
+    culture: PropTypes.string
+  }
+  static defaultProps = {
+    slices: 2,
+    step: 10,
+    isNow: false,
+    showlabels: true,
+    timesliceClassnames: '',
+    culture: 'en'
+  }
+
+  renderSlice(i, classNames, value) {
+    return <SelectableTimeSlice key={i}
+              classNames={classNames}
+              showlabel={this.props.showlabels && !i}
+              format={this.props.timeGutterFormat}
+              culture={this.props.culture}
+              isNow={this.props.isNow}
+              value={value} />
+  }
+
+  renderSlices() {
+    let classNames = this.props.timesliceClassnames
+    switch (this.props.slices) {
```

Comment:
if they aren't used then can we remove them? time slices should still get a prefixed `rbc-time-slice` class or similar regardless I think.